src/index.html

```diff
@@ -13,20 +13,34 @@
     <title>Document</title>
     <link
       rel="stylesheet"
-      href="style.css"
+      href="styles/style.css"
     />
   </head>
-  <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
+  <body class="page">
+    <form
+      action="#"
+      method="get"
+      data-qa="big"
+    >
+      <input
+        class="search search--big page__search"
+        type="text"
+        data-qa="keypress"
+        placeholder="Try “Los Angeles“"
+      />
```

Comment:
The attributes for the `<input>` element starting from line 25 should be each on a new line with 2-space indentation relative to the tag, and the tag’s closing bracket should be on the same level as the opening one to keep the attributes correctly formatted.